docs/how-tos/style-tweaks.md

```diff
@@ -0,0 +1,105 @@
+# Style Tweaks
+
+
+## How to edit LESS and rebuild CSS
+
+To edit LESS you will need to go to the less file you wish to change. The LESS
+files are located in `static/src/.less/`. In this example, we will use
+`custom.less`.
+
+In `custom.less` add the CSS changes you want to make to your site:
+```
+...
+p{
+  font-size: 18;
+  color: red;
+}
+```
+Save `custom.less`.
+
+Before rebuilding the CSS, make sure you have `npm` installed by running the
+following command:
+```
+npm install
+```
+
+To rebuild the site, you will run the following command:
+```
+npm run build && ./manage.py runserver
+```
+The build command will rebuild the CSS and the runserver command allow you to
+view your changes locally.
```

Comment:
I think it's worth mentioning running `npm run watch` in a separate terminal window than `./manage.py runserver` so changes will be rebuilt on save and you can just refresh your browser as you are developing styles.  But then to make a production build, you run `npm run build` before committing.